content/en/security/application_security/setup/aws/lambda/go.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Enabling App and API Protection for AWS Lambda functions in Go
+further_reading:
+    - link: "/security/application_security/how-it-works/"
+      tag: "Documentation"
+      text: "How App and API Protection Works"
+    - link: "/security/default_rules/?category=cat-application-security"
+      tag: "Documentation"
+      text: "OOTB App and API Protection Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting App and API Protection"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "App and API Protection"
+    - link: "https://www.datadoghq.com/blog/datadog-security-google-cloud/"
+      tag: "Blog"
+      text: "Datadog Security extends compliance and threat protection capabilities for Google Cloud"
+---
+
+Configuring App and API Protection (AAP) for AWS Lambda involves:
+
+1. Identifying functions that are vulnerable or are under attack, which would most benefit from AAP. Find them on [the Security tab of your Software Catalog][1].
+2. Setting up AAP instrumentation by using either the [Datadog CLI](https://docs.datadoghq.com/serverless/serverless_integrations/cli), [AWS CDK](https://github.com/DataDog/datadog-cdk-constructs), [Datadog Serverless Framework plugin][2], or manually by using the Datadog tracing layers.
+3. Triggering security signals in your application and seeing how Datadog displays the resulting information.
+
+## Prerequisites
+
+- [Serverless APM Tracing][apm-lambda-tracing-setup] is setup on the Lambda function to send traces directly to Datadog.
+  X-Ray tracing, by itself, is not sufficient for AAP and requires APM Tracing to be enabled.
+
+## Compatibility
+
+**Note**: Threat Protection through Remote Configuration is not supported. Use [Workflows][3] to block IPs in your [WAF][4].
+
+The Datadog Go tracer is distributed as a Go module rather than a Lambda layer. Ensure your functions vendor the latest `github.com/DataDog/datadog-lambda-go` release when enabling AAP.
```

Comment:
```suggestion
The Datadog Go tracer is distributed as a Go module rather than a Lambda layer. Ensure your functions vendor the latest `github.com/DataDog/datadog-lambda-go` release when enabling App and API Protection.
```